core/web/router.go

```diff
@@ -324,9 +325,13 @@ func v2Routes(app chainlink.Application, r *gin.RouterGroup) {
 		authv2.POST("/migrate/:ID", mc.Migrate)
 
 		// PipelineRunsController
+		authv2.GET("/pipeline/runs", paginatedRequest(prc.Index))
```

Comment:
Note: `v2/runs` & `v2/job_spec_errors` are already used by the v1 pipeline. I propose we rename these once v1 is removed.